app/src/main/java/org/kiwix/kiwixmobile/wifi_hotspot/WifiHotspotManager.java

```diff
@@ -0,0 +1,89 @@
+package org.kiwix.kiwixmobile.wifi_hotspot;
+
+import android.content.Context;
+import android.net.wifi.WifiConfiguration;
+import android.net.wifi.WifiManager;
+import android.os.Build;
+import android.os.Handler;
+import android.util.Log;
+import androidx.annotation.NonNull;
+import androidx.annotation.RequiresApi;
+import org.kiwix.kiwixmobile.webserver.ServerStateListener;
+
+/**
+ * WifiHotstopManager class makes use of the Android's WifiManager and WifiConfiguration class
+ * to implement the wifi hotspot feature.
+ * Created by Adeel Zafar on 28/5/2019.
+ */
+
+public class WifiHotspotManager {
+  private final WifiManager wifiManager;
+  private final Context context;
+  WifiManager.LocalOnlyHotspotReservation hotspotReservation;
+  boolean isHotspotEnabled;
```

Comment:
this is only ever written to

Reply:
Remove it. 88c386a8b39dbd16998e5f950cfb5ed0221b0f98